src/ctapipe/monitoring/extractor.py

```diff
@@ -0,0 +1,171 @@
+"""
+Extraction algorithms to compute the statistics from a chunk of images
```

Comment:
Please add a bit more detail so the documentation is useful (this will appear in the SatisticsExtractor page of the API documentation). A new user would have no idea what this class actually does based on the current docstring.

I suggest at minimum:
```py3
"""
Algorithms to compute aggregated time-series statistics from columns of an event table.

These classes take as input an events table, divide it into time chunks, which
may optionally overlap, and computes various aggregated statistics for each
chunk, including the count, mean, median, and standard deviation. The result is
a monitoring table with columns describing the start and stop time of the chunk
and the aggregated statistic values.
"""
```

and maybe also add an example or tutorial.

Reply:
Thanks for the suggestion! While updating this, should I also consider/include the suggested renaming into `StatisticsAggregator`, `aggregator.py`, and accordingly?

Reply:
Yes, I think that would be good to avoid confusing it with ImageExtractor.

Reply:
I updated the docs with your suggestions and renamed `extractor` to `aggregator `. For the tutorial, I planned to put together one once we have implemented the "`TwoPassCalculator`" and the "`FFactor component `".